blogposts/2020/announcing-sourcegraph-3.17.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "Sourcegraph 3.17: Faster and automatic precise code intelligence, preview of code insights, and AND/OR queries for all search modes"
```

Comment:
```suggestion
title: "Sourcegraph 3.17: Faster and automatic precise code intelligence, preview of code insights, and AND/OR queries for searching file contents"
```

Don't feel very strong about this, but think it reads better and also explains it's for file content.

Reply:
i think that's a great improvement! committing the suggestion now and will update later in the post